src/game_server/command.rs

```diff
@@ -41,6 +42,38 @@ impl SerializePacket for CommandOpCode {
     }
 }
 
+#[derive(SerializePacket, DeserializePacket)]
+pub struct BGRA {
```

Comment:
Can we rename this to the more recognizable `Rgba` and change the order of the params in the `new` method to match? Consumers of this struct wouldn't need to know the exact ordering of the colors. They can just put them in standard RGBA order in the constructor and let the serialize/deserialize methods figure out the correct bytes.